xds/src/main/java/io/grpc/xds/internal/rbac/engine/cel/Activation.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.xds.internal.cel;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import javax.annotation.Nullable;
+  
+/**
+ * This is a Java stub for evaluating Common Expression Language (CEL). 
+ * More information about CEL can be found in https://github.com/google/cel-spec. 
+ * Once Java CEL has been open-sourced, this stub will be removed.
+ * 
+ * <p>An object which allows to bind names to values.
+ */
+public abstract class Activation {
+  /** Resolves the given name to its value. Returns null if resolution fails. */
+  @Nullable
+  public abstract Object resolve(String name);
+
+  /** Creates a binder backed up by a map. */
```

Comment:
"backed up" is strange. I'd normally expect "backed," as in the returned Activation is a view of `map`. But that's not what's here. You do mean "backed up" as an copy. Maybe "Creates an immutable binder using the provided mapping." (It is also a _bit_ weird to be calling it "binder" instead of "activation," but whatever.)

Reply:
"Creates an immutable binder using the provided mapping." seems more appropriate here. Thanks for correcting that!